docs/time/index.rst

```diff
@@ -527,13 +573,13 @@ val2
 ^^^^
 
 The ``val2`` argument is available for specialized situations where extremely
```

Comment:
Rereading this feels a bit belittling too - I'd just write "is available for those situations where high ..."